tests/run_test.sh

```diff
@@ -4,6 +4,32 @@
 
 MY_PATH=`pwd`                                   # Capturing the local path of the folder where we are running.
 
+declare -A performanceExpectedTimes
+performanceExpectedTimes["single.point"]=0.014
```

Comment:
What computer system were those reference times run on? What CPU?

Reply:
These tests should be fairly reproducible since they are based on a "relative" time. The ratio between the time for the particular test and a reference test. The test will only fail if there is 
a decrease in the performance (increase of the relative time)

Reply:
And the reference is a collection of CPU intensive problems? Nice :smile:

Just be aware that you might run into false positives (i.e. results that seem slower than 10%) on Travis-CI if the load on the host your instance is running on increases significantly at some point after you took the reference result.

Reply:
Or make it relative to the 1st test!